README.md

```diff
@@ -67,15 +67,15 @@ Build UI files:
   $ ansible-events-ui
 ```
 
-Visit this url: http://localhost:8080/docs#/auth/register_register_api_auth_register_post
+Visit this url: http://localhost/api/docs/auth/register_register_api_auth_register_post
 
 Click "Try it out" on `/api/auth/register`
 
 Change email and password
 
 Click execute
 
-Visit this url: http://localhost:8080/eda
+Visit this url: http://localhost/eda
```

Comment:
Why do you remove the port? this section is for the development env, which it uses that port

Reply:
Same
Both 
http://localhost/eda
and 
http://localhost

will now route you to the UI

Reply:
This section is for the development env, not for docker-compose.

Reply:
This has been reverted in the README